core/test/ragtime/core_test.cljc

```diff
@@ -10,12 +10,12 @@
   (add-migration-id [_ id]
     (swap! data update-in [:migrations] conj id))
   (remove-migration-id [_ id]
-    (swap! data update-in [:migrations] disj id))
+    (swap! data update-in [:migrations] (fn [coll] (filterv #(not= % id) coll))))
   (applied-migration-ids [_]
     (seq (:migrations @data))))
 
 (defn in-memory-db []
-  (InMemoryDB. (atom {:migrations #{}})))
+  (InMemoryDB. (atom {:migrations []})))
```

Comment:
Can you explain this change?

Reply:
The use of `set` which will not preserve the ordering of the migrations. However, functions using `(applied-migration-ids db)` are assuming the ids returned in applied order.

For example, `(rollback-to store index x-id)` rely on the correct ordering of the applied migrations, it simply reverse the applied migration ids, `(down)` it one by one until x-id is found.

Therefore I change `set` to `vector` to keep the ordering.

Reply:
Okay, thanks - that does look like a mistake. However, can you put that fix in another PR? It doesn't seem to be directly related to supporting cljc, but instead seems like a general fix.